mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -141,59 +141,114 @@ def ensure_list(alist):  # {{{
     return alist  # }}}
 
 
-def get_datetimes(ds, timestr, yearoffset):  # {{{
+def parse_time(ds, inTimeVarName, inReferenceDate, outTimeVarName,
+               outReferenceDate):  # {{{
     """
-    Computes a list of datetimes from the time variable in the dataset ds with
-    variable name (or a list of 2 names) given by timestr, typically one of
-    'daysSinceStartOfSim', 'xtime', or ['xtime_start', 'xtime_end'].
+    Converts MPAS times to `netcdftime.DatetimeNoLeap` objects consistent with
+    MPAS's 365-day calendar.
```

Comment:
By 'transient' simulations I mean coupled runs with varying greenhouse gas forcing, for example. So in that case the atmosphere is forced with a varying present-day scenario. I would think in that case we would have to support a gregorian calendar?